src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -237,10 +238,7 @@ def _add_profile_args(parser):
     )
 
     profile_group.add_argument(
-        "-v", action="store_true", required=False, help="Enables verbose mode."
-    )
-
-    profile_group.add_argument(
+        "-v",
```

Comment:
these are 2 different options.
-v -> verbose_mode
--version -> print the version for PA core

Reply:
The help menu options look like this now: 
![image](https://github.com/triton-inference-server/client/assets/58150256/a3fd2dc3-bcb7-48fe-a1d9-fec2954b9d30)